src/lib/common/topology_sampler.cpp

```diff
@@ -164,6 +169,7 @@ static void topology_sample_cb(networkTopology_t *networkTopology) {
         configASSERT(_node_list.last_network_configuration_info.cbor_config_map);
         memcpy(_node_list.last_network_configuration_info.cbor_config_map, cbor_buffer,
                cbor_bufsize);
+        log_cbor_network_configurations(cbor_buffer, cbor_bufsize);
```

Comment:
We will log every time there is a network config change detected by the topology sampler